omero/sysadmins/version-requirements.txt

```diff
@@ -0,0 +1,1883 @@
+********************
+Version requirements
+********************
+
+Proposal for revised minimum version requirements
+-------------------------------------------------
+
+Minimum software versions
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is a proposal to the wider community to solicit their
```

Comment:
With respect to updating the tables to include the information, it will be difficult to fit it in (for the PDF).  The top section (the proposal) was split out from the tables to keep it specific to just the proposed changes for people to look through without being overwhelmed by all the information.  The detailed tables are the source data which was used for the rationale, and the information can certainly be added there if needed, but that's not going to be a summary of the changes as intended which will be easily read and understood by the people we need feedback from.